src/OrchardCore/OrchardCore.ContentManagement/Handlers/ContentsHandler.cs

```diff
@@ -30,7 +30,7 @@ public override Task CreatingAsync(CreateContentContext context)
             var httpContext = _httpContextAccessor.HttpContext;
             if (context.ContentItem.Owner == null && (httpContext?.User?.Identity?.IsAuthenticated ?? false))
             {
-                context.ContentItem.Owner = context.ContentItem.Author = httpContext.User.FindFirstValue(ClaimTypes.NameIdentifier);
+                context.ContentItem.Owner = httpContext.User.FindFirstValue(ClaimTypes.NameIdentifier);
```

Comment:
Reason?

Reply:
The Owner is no longer the same as the Author.

The Author is `User.Name` (set two lines below)
The Owner is ClaimType of NameIdentifier (so UserId)

Reply:
Okay and are we using also ContentItemIndex.Owner the same way?

Reply:
Yes, no change to `ContentItemIndex.Owner` it keeps the same value.

So for an old user / old content item the `Owner` = "admin" (which is now by migration the User.UserId value - for old users)
For a new user -> new content Item the `Owner` = "userguid"